docker-compose.yml

```diff
@@ -2,6 +2,24 @@ version: "3.9"
 
 services:
 
+  test_db:
+    image: postgres:13
+    env_file:
+      - vars.env
+    ports:
+      - "5432:5432"
```

Comment:
Please don't use this port. This is already used by `db` service. This prevents `db` service to start.

Reply:
> This prevents `db` service to start.
That's interesting. What `docker compose` command do you use? I'm asking because the `test_db` service shouldn't be started when `web_server` or `db` service starts.

Reply:
See https://github.com/AlmaLinux/build-system/issues/287